src/robusta/integrations/msteams/msteams_elements/msteams_column.py

```diff
@@ -18,14 +18,14 @@ def __column_set(self) -> map:
     def add_column(
```

Comment:
This file is being renamed. This may constitute an API break, not sure how much that matters.